handle_triggers.go

```diff
@@ -52,6 +62,18 @@ func handleTriggers(
 		)
 	}
 
+	var history = make(map[string]ItemHistory)
```

Comment:
don't use `make`, let's keep it as `map[string]ItemHistory{}`: https://github.com/kovetskiy/zabbixctl/pull/1/files#diff-8f971bdf506d76c9a20a6a4f273bdbd2R159